lib/api/apiUtils/object/coldStorage.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Code based on Yutaka Oishi (Fujifilm) contributions
+ * Date: 11 Sep 2020
+ */
+
+/**
+ * Get response header "x-amz-restore"
+ * Be called by objectHead.js
+ * @param {object} objMD - object's metadata
+ * @returns {string} x-amz-restore
```

Comment:
string or undefined

Reply:
Thanks, I will make sure to udpate this when I will use a proper Arsenal version. I will ping you at that time

Reply:
Done: https://github.com/scality/cloudserver/pull/4504/commits/790dd5a55224adb4489b89e5a22211bb7608e931